src/arbcalls/arb.jl

```diff
@@ -93,8 +93,8 @@ arbcall"void arb_trim(arb_t y, const arb_t x)"
 arbcall"void arb_floor(arb_t y, const arb_t x, slong prec)"
 arbcall"void arb_ceil(arb_t y, const arb_t x, slong prec)"
 #arbcall"void arb_get_fmpz_mid_rad_10exp(fmpz_t mid, fmpz_t rad, fmpz_t exp, const arb_t x, slong n)"
-arbcall"int arb_can_round_arf(const arb_t x, slong prec, arf_rnd_t rnd)"
-arbcall"int arb_can_round_mpfr(const arb_t x, slong prec, mpfr_rnd_t rnd)"
+#arbcall"int arb_can_round_arf(const arb_t x, slong prec, arf_rnd_t rnd)"
```

Comment:
commenting both of them?

Reply:
Instead of choosing which one to keep I just commented out both of them. We can decide later on how we want to handle these types of problems.